### DISCUSSION TITLE
Keep the original date time

### DISCUSSION THREAD

**Discussion by ferbos35** [ImageMagick/ImageMagick#2513] [Help]
Hi everyone,
I was offered this powerful software to manipulate photos using the command line, I can do everything I need, but I need to keep the date and time of the original file after making the various changes.
But I can't find any parameters that can do this.
Or is there a parameter and I don't see it?

I apologize for any "google translator" errors :)

**Comment by snibgo** [comment]
What version of IM, on what platform? What is your command line?

If you use "magick mogrify", insert "-define preserve-timestamp=true". See https://imagemagick.org/script/defines.php

**Comment by fmw42** [comment]
See datetime in the EXIF properties at https://imagemagick.org/script/escape.php. Where do you want to keep the date/time? If you want to keep the date/time in its file creation date, then I do not think ImageMagick convert/magick can do that. Mogrify can on recent versions as described by snibgo above. But you can write the date/time from the EXIF properties to either the comment field or in the output file name.  Alternately, EXIFTOOL, might be able to do what you want. See https://exiftool.org

**Comment by ferbos35** [comment]
Windows 10

7.0.10-25 Q16 x64 2020-08-01

es.

FOR /R T: %%A IN (*.jpg) DO magick mogrify -verbose -resample 72 -quality 100 "%%A"


I have to, starting from a single photo, modify them in a different way.
example one from 300 to 72 dpi One with fixed crop 1.000 * 1.000
Starting from a compound name of the file, which through the program I created in C ++ correctly renames the photos while keeping the creation and modification dates unchanged.
With IM I just have to be able to achieve this.

Note: Various exif already tried, but they don't do image manipulations like I need.

Today I immediately see what snibgo wrote

Thanks for your help
